html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-tall-ref.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<title>Reference for tall legend</title>
+<style>
+body { margin: 0; }
+.legend { position: absolute; height: 160px; margin-left: 20px; inline-size: fit-content; background: lime }
+.fieldset {
+  position: absolute;
+  z-index: -1;
+  margin-top: calc((/* half legend height */ 160px / 2) - (/* half top border */ 20px / 2));
+  background: green;
+  height: 40px;
+  left: 0;
+  right: 0;
+}
+.hello { margin-top: 176px; margin-left: 20px; }
```

Comment:
Assuming that 1em == 16px here? But maybe that's ok?

Reply:
I misunderstood here. It should be 180px, not 176px. I really think we need to figure out why.